proposals/12147-mlmd-removal/protos/artifacts.proto

```diff
@@ -0,0 +1,258 @@
+syntax = "proto3";
+
+service ArtifactService {
+  // Finds all artifacts within the specified namespace.
+  // Namespace field is required. In multi-user mode, the caller
+  // is required to have RBAC verb "list" on the "artifacts"
+  // resource for the specified namespace.
+  rpc ListArtifacts(ListArtifactRequest) returns (ListArtifactResponse) {
+    option (google.api.http) = {
+      get: "/apis/v2beta1/artifacts"
+    };
+  }
+
+  // Finds a specific Artifact by ID.
```

Comment:
I assume it's implied but the subject access review here is the same approach as for other resources (e.g. get on the particular resource name and namespace).